### DISCUSSION TITLE
Could you please provide some instructions on how to load and use this in a browser?

### DISCUSSION THREAD

**Discussion by 1001QAdotNET** [eemeli/yaml#352] [Q&A]
Hi Eemeli

I am new to JavaScript and I would like to load and use this module in a browser
I am trying to understand which module I need to load via <script src="..."></script> in order to be able to use the code in browser side JavaScripts

thanks 
1K1

**Comment by eemeli** [comment]
For that sort of use (i.e. not packaging your own scripts), you'll need to use something like [jsDelivr](https://www.jsdelivr.com/) or [unpkg](https://unpkg.com/) as a CDN. Fair warning though: I've no direct experience using either for `yaml`, so can't really be sure how well they work.
